src/pages/K8s.vue

```diff
@@ -23,6 +23,10 @@
       @updateCPU="handleUpdateCPU"
       @warning="handleWarning"
     />
+    <div id="portWrapper" class="labeled-input">
```

Comment:
We already have the `k8s-restart-required` stuff; maybe we should reuse that instead of having more labels?

Reply:
Thanks for pointing me to it. Works fine.